definitions/checks/system_registration.rb

```diff
@@ -5,15 +5,19 @@ class Checks::SystemRegistration < ForemanMaintain::Check
     tags :default
 
     confine do
-      file_exists?('/etc/rhsm/rhsm.conf') &&
-        !feature(:foreman_server) &&
-        feature(:foreman_proxy)
+      file_exists?('/etc/rhsm/rhsm.conf')
     end
+
+    message <<-MESSAGE
```

Comment:
@iNecas @kgaikwad  Let me know what you think of this new metadata. If it sounds good I'll create a new PR for `message` metadata and not piggyback it with this PR(unless piggybacking is Okay)

Reply:
@swapab,
In my opinion it is good to have but `message` key-name will be very generalized one. 
Naming of this metadata i.e `message` may create a confusion while understanding a purpose of it.

Reply:
@kgaikwad 
Yes, that's right. I thought of it. But couldn't find a suitable name for it.
`failure_message` but the text can be shown for a warning as well as a failure check.
`notification_message` doesn't sound relevant either
`resolution_message` close enough(but not sure if we are showing a resolution or just wanted to tell the administrator something related to failed/warned check)

Any name you have in mind?

Reply:
I'm not sure it's a good place fore metadata. Besides the name, there is also issue when we wanted to update the content on upstream vs. downstream. I would probably like this more passing as argument to `fail!` or `assert` methods

Reply:
A `yaml` file(2 actually one for downstream and one for upstream)? to map failure and/or warning messages per definition.

Reply:
It would mean decoupling the messages from the checks, which would be confusing (at least for me). I would like to keep that in one code.

Reply:
Okay. I have dropped the thought of a new `metadata`. And used existing `fail!` to show the documentation notification message.